docs/fern/features/multimodal/embedding-cache.md

```
@@ -57,6 +60,21 @@ If your workload consists entirely of unique multimodal content, the cache provi
   </Tab>
 </Tabs>
 
+</Tab>
+<Tab title="Intel GPU">
+
+Use the vLLM aggregated cache connector. The connector detects the XPU device and moves cached embeddings between XPU memory and its CPU-side LRU store:
+
+```bash
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)